example/lib/main.dart

```diff
@@ -1 +1,7 @@
-void main() {}
+void info() {
+  final unused = ''
```

Comment:
<table>
  <tbody>
    <tr>
  <td>:rotating_light:</td>
  <td>Expected to find ';'.</td>
</tr>
<tr>
  <td>:speech_balloon:</td>
  <td>The value of the local variable 'unused' isn't used.</td>
</tr>
  </tbody>
</table>